beacons/src/commonMain/kotlin/BeaconInfo.kt

```diff
@@ -18,19 +18,22 @@
 package com.splendo.kaluga.bluetooth.beacons
 
 import com.splendo.kaluga.base.utils.Date
-import com.splendo.kaluga.bluetooth.device.Identifier
 
 typealias BeaconID = Eddystone.UID
 typealias TxPower = Int
 typealias RSSI = Int
 
 data class BeaconInfo(
-    var identifier: Identifier,
+    var identifier: String,
```

Comment:
In case of iOS, `Identifier` is NSUUID. Does it represent a string deep down so no need for `Identifier` at all?

Reply:
Yes, now `identifier` represents Beacon's ID, and not Device's

Reply:
But we should be able to discover different devices even with same beacon id, I'll fix it, thanks for pointing out